### PR TITLE
Reactivate LTI Upload Tool

### PR DESCRIPTION
**Pull request by albertliangcode**
Disabled HTTPS on devstack.
Found no problems when running LTI Upload Tool locally.

### CODE REVIEW

**Comment by albertliangcode** [comment]
Please review, @stvstnfrd!

**Comment by albertliangcode** [comment]
Okay!  Please review when possible, @stvstnfrd @dcadams!

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 SITE_NAME = 'localhost:8000'
 # By default don't use a worker, execute tasks as if they were local functions
 CELERY_ALWAYS_EAGER = True
+HTTPS = 'off'
```
Comment (stvstnfrd) [inline lms/envs/devstack.py]: Looking at the other settings files, we probably want to make the corresponding changes to `cms`, as well.

Can you add this override to `cms/envs/dev.py` and `cms/envs/devstack.py`?

**Comment by stvstnfrd** [comment]
@albertliangcode @mondiaz Can you refresh my memory about what the original issue was? I know that  @albertliangcode 's change makes sense as a fix for devstack, but I'm unclear as to how/why this will work when we put it back on stage, despite no relevant changes made.

Can you provide some context surrounding the original issue?

File: lms/envs/devstack.py (hunk 1 above)
Comment (albertliangcode) [inline lms/envs/devstack.py]: Sure!  I'll make the change both here and on the upstream PR.

Comment (stvstnfrd) [inline lms/envs/devstack.py]: Not a big deal, but in the future, it's usually easier to wait until we merge the feature to our remote before opening a PR upstream. That way we can address any/all issues with our internal review before potentially receiving duplicate feedback from an upstream reviewer.

This approach saves you and the reviewers time. Additionally, we do want this feature merged to origin soon-ish, but we aren't in any real rush to push it upstream. We're doing that as a "favor" as good community members, but we don't really care if it takes a long time to merge there.

Also, when you open a PR upstream, please tag the Stanford reviewer who looked at the code on our end. That way, we can weigh in on feedback that may be provided from edX.

Comment (albertliangcode) [inline lms/envs/devstack.py]: @stvstnfrd: Should I squash these HTTPS changes into one commit?  Or should I leave them as two commits - one for LMS and one for CMS?

Comment (albertliangcode) [inline lms/envs/devstack.py]: And okay!  I'll do that on the upstream PR.

Comment (stvstnfrd) [inline lms/envs/devstack.py]: Yes, squash the 2 `envs` commits together, but keep Azim's separate.

**Comment by albertliangcode** [comment]
From what I understand, when Azim first added the LTI Tool in late April, everything was working fine.  It wasn't until about a month later that something broke, and the tool was taken down.

The issue was that even if the user correctly inputted the key and secret, the authentication failed.  Monica and I couldn't reproduce the error locally on either master or prod code.  Because of this, we wanted to see if there was something different between local and stage that would cause the tool to work fine on devstack, but fail on stage.

**Comment by albertliangcode** [comment]
@stanford-online-robot, DON'T FAIL ME NOW, I WANT YOU TO TEST THIS PLEASE!!!!!

**Comment by albertliangcode** [comment]
Okay robot, enough games.  Seriously, test this please.

**Comment by stvstnfrd** [comment]
@albertliangcode Per previous conversation, this seems good to go ahead and merge. Once we get it onto stage, we'll want to confirm the correct behavior still holds.

**Comment by albertliangcode** [comment]
Okay!  Will merge now!

**Comment by albertliangcode** [comment]
@stvstnfrd: Merged to master, but will not delete origin/aliang8/lti-ut branch until Stage checks have passed.

**Comment by albertliangcode** [comment]
Branch deleted!

**Pull request merged**